container-setup/install/install-ocm.sh

```diff
@@ -8,6 +10,19 @@ fi
 echo "in container";
 source /container-setup/install/helpers.sh
 
-remove_coloring go get -v -u github.com/openshift-online/ocm-cli/cmd/ocm;
-ln -s /root/go/bin/ocm /usr/local/bin/ocm;
+LATEST_RELEASE="https://github.com/openshift-online/ocm-cli/releases/latest/download/ocm-linux-amd64"
+
+pushd /usr/local/bin/
```

Comment:
Not opposed to your idea, but usually on the repo we put each tool.in its own folder

Reply:
Amended to install to /usr/local/ocm, symlink to /usr/local/bin/ocm